### PR TITLE
Add identifier component

### PR DESCRIPTION
**Pull request by mejiaj**
**Introducing the USWDS Identifier.** The Identifier is a sub-footer component for the bottom of any government webpage. It's a complement to the security-focused Banner: it both identifies the site domain and its parent department or agency, and provides the links required on any federal website. 

## Description

Adds identifier component.
[USWDS site preview](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/add-identifier/components/identifier/)

#### [Component](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/jm-component-anchor/components/detail/identifier--default.html)

Screen Shot 2020-09-15 at 11 35 18 AM

[v0.0.1](https://user-images.githubusercontent.com/3385219/90673748-c8dbd300-e21d-11ea-86e2-dbc0e8fb6e53.png)

## Additional information
~Currently all possible settings are in the `_anchor.scss` while we iterate and select the best defaults.~

Settings have been updated and component renamed to **Identifier**.

Before you hit Submit, make sure you’ve done whichever of these applies to you:

- [x] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [x] Run `npm test` and make sure the tests for the files you have changed have passed.
- [x] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [x] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

### CODE REVIEW

File: src/components/identifier/identifier.njk [hunk 1]
```diff
@@ -1,12 +1,84 @@
...
-<div class="padding-2">
-  <h4>Multiple logos identifier</h4>
-    {%- render '@identifier--multiple-logos' -%}
-</div>
-<div class="padding-2">
-  <h4>No logos identifier</h4>
-  {%- render '@identifier--no-logos' -%}
+{% set taxpayer_disclaimer %}Produced and published at taxpayer expense.{% endset %}
+
+<div class="usa-identifier">
```
Comment (mejiaj) [inline src/components/identifier/identifier.njk:3]: Is there a specific reason for changing identifier from `<section> → <div>`?

Comment (thisisdano) [reply src/components/identifier/identifier.njk:3]: Wasn't sure if these met the definition of a section — specifically that they do not begin with a heading or might be considered "chapters"

Comment (mejiaj) [reply src/components/identifier/identifier.njk:3]: Voiceover with landmark browsing 
- Enable voiceover `cmd + F5`
- Enable rotor `ctrl + opt + u`
- Right/Left arrow to find landmarks

---

## Variations

### ❌ Current

image

### ❌ Div w/ aria-label
image

### ✅ Section w/ aria-label
image

Comment (mejiaj) [reply src/components/identifier/identifier.njk:3]: I now see three areas with the new section updates:
image

**Review by thisisdano — APPROVED** [review]

File: src/components/identifier/identifier.njk (hunk 1 above)
Comment (thisisdano) [reply src/components/identifier/identifier.njk:3]: OK — I think that's where we want it.

**Comment by thisisdano** [comment]
Gonna pull the most recent work into a site branch and finish QA.

**Comment by thisisdano** [comment]
This is pending final final Spanish copy, but this PR is probably good to review and merge now, and we can update any copy in a subsequent PR.

**Comment by thisisdano** [comment]
@lggsa Here are all the Spanish variants. Do these look the way you expect?

[default](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/jm-component-anchor/components/detail/identifier--lang-es.html) →
[multiple agencies](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/jm-component-anchor/components/detail/identifier--multiple-logos-lang-es.html) →
[taxpayer disclaimer](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/jm-component-anchor/components/detail/identifier--taxpayer-disclaimer-lang-es.html) →

**Pull request merged**